documentation/API_documentation/QoD_Stable_Bandwidth_API.md

```diff
@@ -0,0 +1,180 @@
+# Overview
+
+<span class="colour" style="color:rgb(0, 0, 0)">The Quality-On-Demand (QoD) API for stable throughput provides programmable interface for developers and other users (capabilities consumers) to specify stable data-transfer throughput managed by Telco networks without necessity to have in-depth knowledge of the 4G/5G system complexity by abstracting the internal complexity of telecom systems [1].</span>
```

Comment:
"Telco networks without necessity" -> "Telco networks, without necessity"

Reply:
Done in [1fd0cfe](https://github.com/camaraproject/QualityOnDemand/pull/17/commits/1fd0cfe1c718e6362cfb0f1536790f28fb003d8e)